### DISCUSSION TITLE
No audio output through spotify connect

### DISCUSSION THREAD

**Discussion by bobbyb12** [rern/rAudio#291] [Bug Reports]
Hello,
my setup is:
rAudio 1 • 20210403
5.10.25-4-ARCH armv6l
Raspberry Pi Model B Rev 2
BCM2835 • 455MiB
ARM1176 @ 700MHz
connected through ethernet

I can connect through spotify connect to my pi but when the music is playing, I have no audio output on my headphone jack.
webradio and upnp is working fine. Tried to play to spotify through different clients with no success. How can I troubleshoot my issue?

Thanks

**Comment by madcgm** [comment]
My setup:
rAudio 1 • 20210502
5.10.27-3-ARCH armv6l
Raspberry Pi Model B Rev 2
BCM2835 • 455MiB
ARM1176 @ 700MHz

I have the very same problem. My spotify app can see the device Spotifyd@rAudio, and when Iconnect to it, the Spotify app shows that the song is being played on tthat device, but there is no sound from the headphone jack of the Raspberry.

This is the message I get:

May 04 19:19:05 rAudio spotifyd[6447]: No password_cmd specified
May 04 19:19:05 rAudio spotifyd[6447]: No proxy specified
May 04 19:19:05 rAudio spotifyd[6447]: Using software volume controller.
May 04 19:19:06 rAudio systemd[1]: Started A spotify playing daemon.
May 04 20:08:58 rAudio spotifyd[6447]: Connecting to AP "gew1-accesspoint-e-k177.ap.spotify.com:443"
May 04 20:08:59 rAudio spotifyd[6447]: Authenticated as "madcgm" !
May 04 20:08:59 rAudio spotifyd[6447]: Country: "ES"
May 04 20:08:59 rAudio spotifyd[6447]: Using alsa sink
May 04 20:09:00 rAudio spotifyd[6447]: Loading  with Spotify URI 
May 04 20:09:00 rAudio spotifyd[6447]: Caught panic with message: attempted to zero-initialize type `librespot_tremor::tremor_sys::ov_callbacks`, which is invalid

How can I solve the problem...?
Thanks in advance.
